File: 251003/zdanovich/Publisher/src/main/java/by/zdanovich/Publisher/models/Writer.java
package by.zdanovich.Publisher.models;

import jakarta.persistence.*;
import lombok.Data;

import java.io.Serializable;


@Entity
@Table(name = "tbl_writer")
@Data
public class Writer implements Serializable {
    @Id
    @Column(name = "id")
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private long id;

    @Column(name = "login", unique = true)
    private String login;

    @Column(name = "password")
    private String password;

    @Column(name = "firstname")
    private String firstname;

    @Column(name = "lastname")
    private String lastname;
}

File: 251001/Smychok/LAB4/LAB4/Models/Writer.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace LAB4.Models;

public class Writer
{
    public int id { get; set; }
    
    [Column(TypeName = "varchar(64)")]
    public string login { get; set; }
    
    [Column(TypeName = "varchar(128)")]
    public string password { get; set; }
    
    [Column(TypeName = "varchar(64)")]
    public string firstname { get; set; }
    
    [Column(TypeName = "varchar(64)")]
    public string lastname { get; set; }
    
    public List<Issue> issues { get; set; } = new();
}